src/extra/components/utils.py

```diff
@@ -7,6 +7,33 @@
     3: {'SA3', 'LA3', 'SCS', 'SQS', 'SXP'}
 }
 
+
+def _identify_instrument(data, threshold=0.5):
```

Comment:
Looks like this function is completely uncovered by tests. It would be nice to exercise the most-common-prefix pathway, directly or indirectly. It doesn't need to be in this PR, though.

Reply:
Done in 7406154abad848374b7aecfacaf2a6298573c2b5